src/test/java/com/googleinterns/zoomtube/servlets/LectureServletTest.java

```diff
@@ -83,11 +82,40 @@ public void tearDown() {
     testServices.tearDown();
   }
 
+  @Test
+  public void doPost_missingName_badRequest() throws IOException, ServletException {
+    when(request.getParameter(LectureServlet.PARAM_LINK)).thenReturn(TEST_LINK);
+
+    servlet.doPost(request, response);
+
+    verify(response).sendError(HttpServletResponse.SC_BAD_REQUEST, "Missing name parameter.");
```

Comment:
nit: please make the error message constants

Reply:
Done, thanks!